docs/Validation.md

```diff
@@ -1,296 +1,318 @@
+    xsi:schemaLocation="htp://foo foo.xsd">
+  <bar />
+</foo>
+```
+
+### xml-model with XSD
+
+To bind the XML file `foo.xml` to the XSD file `foo.xsd`, you can use [xml-model](https://www.w3.org/TR/xml-model/) processing instruction strategy like this:
+
+```xml
+<?xml-model href="foo.xsd" ?>
+<foo>
+  <bar />
+</foo>
+```
+
+### XML catalog with XSD
+
+Using an XML catalog can be helpful when you wish to use elements from a namespace but  don't want to set the real path of the XSD file for each XML document.
+
+In other words you want to write this XML file:
+
+
+```xml
+<foo xmlns="http://foo" >
+  <bar />
+</foo>
+```
+
+and validate it with the `foo.xsd` XSD file:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    elementFormDefault="qualified"
+    targetNamespace="http://foo">
+  <xs:element name="foo">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element name="bar" />
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+</xs:schema>
+```
+
+To do this, you need to write an XML catalog, and an entry for the XSD. In this example, we'll name the catalog `catalog.xml`. The catalog looks like this:
+
+```xml
+<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+
+  <uri
+      name="http://foo"
+      uri="./foo.xsd" />
+
+</catalog>
+```
+
+You must register your `catalog.xml` in the vscode `settings.json` like this:
+
+```json
+"xml.catalogs": [
+   "catalog.xml"
+]
+```
+
+You can also register the catalog using its absolute path:
+
+| Windows                            | MacOS                        | Linux                       |
+| ---------------------------------- | ---------------------------- | --------------------------- |
+| `C:\\Users\\path\\to\\catalog.xml` | `/Users/path/to/catalog.xml` | `/home/path/to/catalog.xml` |
+
+Please note that you can declare your catalog path with `file:///` if you need it.
+
+### XML file association
+
+You can use the XML file association strategy to bind the XML file `foo.xml` with the XSD file `foo.xsd` by adding the following into your vscode `settings.json`:
+
+```json
+"xml.fileAssociations": [
+   {
+       "pattern": "foo.xml",
+       "systemId": "foo.xsd"
+   }
+]
+```
+
+Please note that you can use wildcards in the pattern (ex: `foo*.xml`):
+
+```json
+"xml.fileAssociations": [
+   {
+       "pattern": "foo*.xml",
+       "systemId": "foo.xsd"
+   }
+]
+```
+
+In this case, all XML files that start with foo and end with .xml will be associated with the XSD (foo1.xml, foo2.xml, etc)
+
+## Validation with DTD grammar
+
+To associate your XML with a DTD grammar you can use several strategies:
+
+ * [DOCTYPE](#DOCTYPE)
+ * [xml-model with DTD](#xml-model-with-DTD) processing instruction
+ * [XML catalog with DTD](#XML-catalog-with-DTD)
+ * XML file association is not supported for DTD at the moment. Please see [issue 184](https://github.com/redhat-developer/vscode-xml/issues/184))
```

Comment:
This feature is not supported, please add a section with sample with this feature.